site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1088,6 +1088,8 @@ private function updateGradeable(Gradeable $gradeable, $details) {
             }
         }
 
+        $regrade_modified = false;
```

Comment:
Is there any particular reason why this is defined here, ~50 lines from where it is next used?

Reply:
This is the closest out of for loop scope location. I wish it could be closer but I don't want to repeatedly set the variable and want it to have a default value of false.

Reply:
Oh yeah, I see now.  Perhaps it should have a comment indicating what purpose it serves?

Reply:
Sure I'll go ahead and throw that in.